### PR TITLE
Add proxy detection

### PR DESCRIPTION
**Pull request by BrockBeaudry**
* Adds detection for the Proxy object

https://caniuse.com/#feat=proxy

### CODE REVIEW

File: feature-detects/proxy.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*!
+{
+  "name": "Proxy Object",
+  "property": "Proxy",
+  "caniuse": "proxy",
```
Comment (rejas) [inline feature-detects/proxy.js]: you can add a authors tag here with you name/userhandle if you want :-)

File: feature-detects/proxy.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "tags": ["dom"],
```
Comment (rejas) [inline feature-detects/proxy.js]: might be an idea to put it in the dom directory if it has that tag?

File: feature-detects/proxy.js (hunk 1 above)
Comment (BrockBeaudry) [reply feature-detects/proxy.js]: Added, thanks for the suggestion

File: feature-detects/proxy.js (hunk 2 above)
Comment (BrockBeaudry) [reply feature-detects/proxy.js]: I don't think it actually has this tag so I went ahead and removed it

**Comment by rejas** [comment]
Thx for the PR.

**Pull request merged**